packages/gatsby-plugin-image/src/components/gatsby-image.browser.tsx

```diff
@@ -215,4 +217,19 @@ export const GatsbyImage: FunctionComponent<GatsbyImageProps> = function GatsbyI
 ) {
   return <GatsbyImageHydrator {...props} />
 }
+
+export const propTypes = {
+  image: PropTypes.object.isRequired,
+  alt: ((props: GatsbyImageProps, ...rest): Error | undefined => {
+    if (!props.alt && props.alt !== ``) {
+      return new Error(
+        `The "alt" prop is required for GatsbyImage. If the image is purely presentational then pass an empty string: e.g. alt="". Learn more: https://a11y-style-guide.com/style-guide/section-media.html`
```

Comment:
So does this currently only validate `GatsbyImage` or also `StaticImage`? If both, maybe an "and" or `gatsby-plugin-image`?

Reply:
StaticImage has separate validation, but that reminds me that I should update that too.